src/main/java/seedu/address/ui/HelpDeleteWindow.java

```diff
@@ -0,0 +1,106 @@
+package seedu.address.ui;
+
+import java.util.logging.Logger;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.input.Clipboard;
+import javafx.scene.input.ClipboardContent;
+import javafx.stage.Stage;
+import seedu.address.commons.core.LogsCenter;
+
+/**
+ * Controller for a help page
+ */
+public class HelpDeleteWindow extends UiPart<Stage> {
+
+    public static final String USERGUIDE_URL = "https://ay2324s2-cs2103t-w10-2.github.io/tp/UserGuide.html";
+    public static final String HELP_MESSAGE = "Deletes the specified person from the Pooch Planner"
+            + "\n" + "" + "\n"
+            + "Format: /delete ; name : [name] "
+            + "\n" + "" + "\n"
+            + "Go to our UG for more information : " + USERGUIDE_URL;
+
+    private static final Logger logger = LogsCenter.getLogger(HelpWindow.class);
+    private static final String FXML = "HelpWindow.fxml";
+
+    @FXML
+    private Button copyButton;
+
+    @FXML
+    private Label helpMessage;
+
+    /**
+     * Creates a new HelpWindow.
+     *
+     * @param root Stage to use as the root of the HelpWindow.
+     */
+    public HelpDeleteWindow(Stage root) {
+        super(FXML, root);
+        helpMessage.setText(HELP_MESSAGE);
+    }
+
+    /**
+     * Creates a new HelpWindow.
+     */
+    public HelpDeleteWindow() {
+        this(new Stage());
+    }
+
+    /**
+     * Shows the help window.
+     * @throws IllegalStateException
+     *     <ul>
+     *         <li>
+     *             if this method is called on a thread other than the JavaFX Application Thread.
+     *         </li>
+     *         <li>
+     *             if this method is called during animation or layout processing.
+     *         </li>
+     *         <li>
+     *             if this method is called on the primary stage.
+     *         </li>
+     *         <li>
+     *             if {@code dialogStage} is already showing.
+     *         </li>
+     *     </ul>
+     */
+    public void show() {
+        logger.fine("Showing help page about the application.");
```

Comment:
Maybe abstract the magic string out as a constant variable?

Reply:
i agree!! i have added a variable